syntax="proto2";

option optimize_for=SPEED;

package aap_protobuf.service.radio;

enum RadioMessageId {
  RADIO_MESSAGE_ACTIVE_RADIO_NOTIFICATION = 32769;
  RADIO_MESSAGE_SELECT_ACTIVE_RADIO_REQUEST = 32770;
  RADIO_MESSAGE_STEP_CHANNEL_REQUEST = 32771;
  RADIO_MESSAGE_STEP_CHANNEL_RESPONSE = 32772;
  RADIO_MESSAGE_SEEK_STATION_REQUEST = 32773;
  RADIO_MESSAGE_SEEK_STATION_RESPONSE = 32774;
  RADIO_MESSAGE_SCAN_STATIONS_REQUEST = 32775;
  RADIO_MESSAGE_SCAN_STATIONS_RESPONSE = 32776;
  RADIO_MESSAGE_TUNE_TO_STATION_REQUEST = 32777;
  RADIO_MESSAGE_TUNE_TO_STATION_RESPONSE = 32778;
  RADIO_MESSAGE_GET_PROGRAM_LIST_REQUEST = 32779;
  RADIO_MESSAGE_GET_PROGRAM_LIST_RESPONSE = 32780;
  RADIO_MESSAGE_STATION_PRESETS_NOTIFICATION = 32781;
  RADIO_MESSAGE_CANCEL_OPERATIONS_REQUEST = 32782;
  RADIO_MESSAGE_CANCEL_OPERATIONS_RESPONSE = 32783;
  RADIO_MESSAGE_CONFIGURE_CHANNEL_SPACING_REQUEST = 32784;
  RADIO_MESSAGE_CONFIGURE_CHANNEL_SPACING_RESPONSE = 32785;
  RADIO_MESSAGE_RADIO_STATION_INFO_NOTIFICATION = 32786;
  RADIO_MESSAGE_MUTE_RADIO_REQUEST = 32787;
  RADIO_MESSAGE_MUTE_RADIO_RESPONSE = 32788;
  RADIO_MESSAGE_GET_TRAFFIC_UPDATE_REQUEST = 32789;
  RADIO_MESSAGE_GET_TRAFFIC_UPDATE_RESPONSE = 32790;
  RADIO_MESSAGE_RADIO_SOURCE_REQUEST = 32791;
  RADIO_MESSAGE_RADIO_SOURCE_RESPONSE = 32792;
  RADIO_MESSAGE_STATE_NOTIFICATION = 32793;
}